hack/perf-ibmcloud-openshift.sh

```diff
@@ -122,6 +117,16 @@ REGION=${REGION}
 EOM
 infomsg "==END CONFIG=="
 
+install_istio() {
+  if [ -z "${ISTIO_VERSION}" ]; then
+    hack/istio/download-istio.sh -iv ${ISTIO_VERSION}
+  else
+    # Download latest
+    hack/istio/download-istio.sh
+  fi
+  hack/istio/install-istio-via-istioctl.sh -iee true -cp openshift
```

Comment:
Just FYI: this isn't guaranteed to use the version passed in and set in ISTIO_VERSION.
Presumably, this allows you to test with an earlier version of Istio... but if you already had installed 1.14 and you pass in `-iv 1.13` this won't do what you think it will do.

Reply:
https://github.com/kiali/kiali/blob/v1.52.0/hack/istio/install-istio-via-istioctl.sh#L244-L245

Reply:
a hacky workaround would be:

`touch _output/istio-${ISTIO_VERSION}`

but then everytime the user runs install-istio-via-istioctl, it will install THAT version.

I think we should update install-istio-via-istictl so it can take a `--istio-version` option.